src/pages/example.tsx

```diff
@@ -1,16 +1,81 @@
 import { FaArrowLeft } from 'react-icons/fa'
 
-import { PageWrapper, Link, Button } from '@/components'
+import { PageWrapper, Link, Button, Tooltip } from '@/components'
 import { withMakaira } from '@/makaira/withMakaira'
 
 export default function Example() {
   return (
     <PageWrapper title="Example page" prefix="You are looking at">
-      <Link pathname="/">
-        <Button icon={FaArrowLeft} variant="primary" iconPosition="left">
+      <div>
+        <Link pathname="/">
+          <Button icon={FaArrowLeft} variant="primary" iconPosition="left" loading>
+            Back to homepage
+          </Button>
+        </Link>
+        <Button icon={FaArrowLeft} variant="secondary" iconPosition="right">
           Back to homepage
         </Button>
-      </Link>
+        <Button icon={FaArrowLeft} variant="reduced" iconPosition="left">
+          Back to homepage
+        </Button>
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="right">
+          Button icon right
+        </Button>
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" loading>
+          Button loading
+        </Button>
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" disabled>
+          Button Disabled
+        </Button>
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" level={-1}>
+          Button level -1
+        </Button>
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" level={0}>
+          Button level 0
+        </Button>
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" level={1}>
+          Button level 1
+        </Button>
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" />
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" loading />
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" disabled />
+
+        <Button icon={FaArrowLeft} variant="primary" iconPosition="left" tooltip='An additional info tooltip is displayed'>
+          Button with tooltip
+        </Button>
+      </div>
+      <br />
+      <br />
+      <Tooltip
+        placement="top"
+        overlay={<span>An additional info tooltip is displayed</span>}
+      >
+        <button>Back to homepage</button>
```

Comment:
Please switch these button to our buttons and not the default browser ones. Looks weird if there some unstyled buttons.